doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -169,18 +169,22 @@ async def _start_build(self, metadata: ImageMetadata, build_repo: BuildRepo, dyn
             raise ValueError(f"[{metadata.distgit_key}] Dockerfile must have a '__doozer_uuid_tag' environment variable; Did you forget to run 'doozer beta:images:konflux:rebase' first?")
 
         # Ensure the Application resource exists
-        app_name = self._config.group_name.replace(".", "-")
+        group_name = self._config.group_name.replace(".", "-")
+        app_name = group_name
+        app_name = f"{app_name}-{metadata.runtime.assembly}"
```

Comment:
use group_name directly

Reply:
I'm using the group name below: https://github.com/openshift-eng/art-tools/pull/1105/files#diff-e45d70130344e3d45bd59cb091bd0a794121c675ce90f17c5b4e4702ded620ffR183

So didn't want to replace the `.` with `-` again